packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/package-files/package.json

```diff
@@ -0,0 +1,133 @@
+{
+  "name": "@azure/load-testing",
+  "version": "1.0.0-beta.1",
+  "description": "\"This package contains Microsoft Azure LoadTestingClient client library.\"",
```

Comment:
extra double quotes?

Reply:
Thanks for found this, it should have been resolved :)